v2/api/containerinstance/v1beta20211001/container_group_types_gen.go

```diff
@@ -7149,10 +6782,130 @@ func (view *ContainerProperties_InstanceView_STATUS) AssignProperties_To_Contain
 	return nil
 }
 
+// Generated from: https://schema.management.azure.com/schemas/2021-10-01/Microsoft.ContainerInstance.json#/definitions/EnvironmentVariable
+type EnvironmentVariable struct {
+	// +kubebuilder:validation:Required
+	// Name: The name of the environment variable.
+	Name *string `json:"name,omitempty"`
+
+	// SecureValue: The value of the secure environment variable.
+	SecureValue *string `json:"secureValue,omitempty"`
```

Comment:
I think this should be a secret ref, like it was in the `ContainerGroup_Properties_Containers_Properties_EnvironmentVariables_Spec` type.

Reply:
This is _also_ flagged as `x-ms-secret` in the Swagger, [see](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2021-10-01/containerInstance.json#L1737)

So for some reason neither this nor the above workspace key are getting correctly set after the submodule update. Possibly because we've just updated the Azure REST API Specs (Swagger) submodule here but not the Azure Resource Manager Schema too? I think we might need to update them both (in sync). These types seem to have been renamed because of a change in the Swagger, but if we don't also update Resource Manager Schema then it doesn't have the new names so the type-merging doesn't work? (Just thinking out loud here but it seems plausible to me)

Reply:
Fixed submodule reference and this change went away.